frontend/src/components/applications/ApplicationNavBar.vue

```diff
@@ -136,6 +148,8 @@ export default {
       switch (item.routeName) {
         case APPLICATION_ROUTES.FACILITY_DETAILS:
           return this.isFacilityDetailsComplete ? 'mdi-check-circle' : 'mdi-circle'
+        case APPLICATION_ROUTES.ELIGIBILITY:
+          return this.isEligibilityComplete ? 'mdi-check-circle' : 'mdi-circle'
```

Comment:
Not sure if you want to refactor the logic to get the correct icon into it's own function? It's always the same pair of icons so it would save some code. I know you like less code!

Reply:
yes, I love clean code! I think it looks cleaner now. thanks Wes!